src/widgets/mentors/ui/mentors.module.scss

```diff
@@ -0,0 +1,33 @@
+.mentors-container {
+  background: $color-gray-100;
+}
+
+.mentors-content {
+  flex-direction: row;
+
+  @include media-tablet-large {
+    flex-direction: column;
+    gap: 40px;
+  }
+}
+
+.mentors-info {
+  display: flex;
+  flex-direction: column;
+  max-width: 640px;
+
+  @include media-tablet-large {
+    width: 100%;
+  }
+}
```

Comment:
Let's nest the classes like in other components:
```
.mentors-container {
   .mentors-content {
     .mentors-info {
     }
   }
}
```

Reply:
Done